src/app/core/services/repo-creator.service.ts

```diff
@@ -0,0 +1,41 @@
+import { Injectable } from '@angular/core';
+import { flatMap } from 'rxjs/operators';
+import { Observable, of, pipe, UnaryFunction } from 'rxjs';
+import { GithubService } from './github.service';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class RepoCreatorService {
+  constructor(
+    private githubService: GithubService
+  ) {}
+
+  /**
+   * Checks if the necessary repository has been created.
+   */
+  public verifyRepoCreation(phaseOwner: string, phaseRepo: string): UnaryFunction<Observable<boolean | null>, Observable<boolean>> {
+    return pipe(
+      flatMap((isFixAttempted: boolean | null) => {
+        if (isFixAttempted === null) {
```

Comment:
Is it possible to change this expression to `!isFixAttempted` instead? Or would it cause any problems?